client/transport/build.gradle

```diff
@@ -55,3 +55,15 @@ namingConventions {
   //we don't have integration tests
   skipIntegTestInDisguise = true
 }
+
+
+javadoc {
+  dependsOn ':core:javadoc'
+  String host;
+  if (version.endsWith("-SNAPSHOT")) {
+    host = "https://snapshots.elastic.co";
+  } else {
+    host = "https://artifacts.elastic.co";
+  }
+  options.linksOffline host + "/javadoc/org/elasticsearch/elasticsearch/${version}", "../../core/build/docs/javadoc/"
```

Comment:
I don't mean order of dependencies. I mean order of linksOffline options. In my experience that counts, no matter if the deps were properly built. If org.elasticsearch.* is first, the links to the low-level client won't work. At least that is what I saw.